templates/cisco_nxos_show_ip_interface_brief.template

```diff
@@ -0,0 +1,9 @@
+Value Required INTF (\S+)
+Value Required IPADDR ([a-zA-Z0-9./]+)
+Value STATUS (\S+-\S+)
+Value LINK (\S+-\S+)
+Value PROTO (\S+-\S+)
+
+Start
+  ^${INTF}\s+${IPADDR}\s+${STATUS}/${LINK}/${PROTO} -> Record
```

Comment:
I think ${STATUS} and ${PROTO} need to be swapped around, but other than that everything looks good.

Reply:
Updated!